tests/aws/services/events/test_events_cross_account_region.py

```diff
@@ -0,0 +1,269 @@
+import json
+import time
+
+from localstack.testing.aws.util import is_aws_cloud
+from localstack.testing.pytest import markers
+from localstack.utils.strings import short_uid
+from tests.aws.services.events.helper_functions import sqs_collect_messages
+from tests.aws.services.events.test_events import EVENT_DETAIL, TEST_EVENT_PATTERN_NO_SOURCE
+
+SOURCE_PRIMARY = "source-primary"
+SOURCE_SECONDARY = "source-secondary"
+
+
+class TestEventCrossRegion:
+    @markers.aws.validated
+    def test_event_bus_to_event_bus_cross_region(
+        self, region_name, secondary_region_name, aws_client_factory, cleanups, snapshot
+    ):
+        # overwriting randomized region https://docs.localstack.cloud/contributing/multi-account-region-testing/
+        # requires manually adding region replacement for snapshot
+        snapshot.add_transformer(snapshot.transform.regex(region_name, "<region-name>"))
+        snapshot.add_transformer(
+            snapshot.transform.regex(secondary_region_name, "<region-name-secondary>")
+        )
+
+        aws_client_primary = aws_client_factory(region_name=region_name)
+        aws_client_secondary = aws_client_factory(region_name=secondary_region_name)
+
+        # Create event buses
+        event_bus_name_primary = f"test-event-bus-primary-{short_uid()}"
+        aws_client_primary.events.create_event_bus(Name=event_bus_name_primary)["EventBusArn"]
```

Comment:
any particular reason why the primary bus is not saved not variable like the secondary?

Reply:
the arn is not needed